### PR TITLE
BE-88 Implement i18n of endpoint responses

### PR DESCRIPTION
**Pull request by DeMensa123**

### CODE REVIEW

File: antiplag/locale/sk/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
```
Comment (OliverKanat) [inline antiplag/locale/sk/LC_MESSAGES/django.po:6]: Are the comments needed? Can we delete them?

File: antiplag/locale/sk/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+"POT-Creation-Date: 2021-03-19 20:04+0100\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=4; plural=(n % 1 == 0 && n == 1 ? 0 : n % 1 == 0 && n "
+">= 2 && n <= 4 ? 1 : n % 1 != 0 ? 2: 3);\n"
```
Comment (OliverKanat) [inline antiplag/locale/sk/LC_MESSAGES/django.po:20]: Can you please fill this out?

**Review by OliverKanat — CHANGES_REQUESTED** [review]

File: antiplag/serializers.py [hunk 3]
```diff
@@ -64,4 +64,4 @@ class DocumentResultSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Document
-        fields = ("name", "result")
+        fields = ("name", "result", "text")
```
Comment (OliverKanat) [inline antiplag/serializers.py:67]: Is this part of task's scope?

Comment (DeMensa123) [reply antiplag/serializers.py:67]: Nop, it's in the scope of this task: https://p3k-team.atlassian.net/browse/BE-80?atlOrigin=eyJpIjoiMjkwNGZmZTU2MzNhNDA2YmIzNmEyMTA3YjlmMjc4YjAiLCJwIjoiaiJ9

Task's scope is only to provide document content which is 1 word, therefore I did it as a part of task BE-88.

Comment (OliverKanat) [reply antiplag/serializers.py:67]: Pls move it to its own branch. Even though it's such a small task.

**Comment by MCFreddie777** [comment]
Just curious:
 - Does it work now? 
 - In which file is the current language set in? 
 - How are we able to change english into slovak and back?

**Comment by DeMensa123** [comment]
It depends on user's browser settings. It checks accept language header so if in accept language is sk, that would mean that if there is slovak language version, it will be provided. If not, it will give english version. 
So If user has Slovak as local language in settings, then he will see slovak translations. Otherwise, default will be en-us. I've tried to change language settings in browser and it worked as I wrote above.

The current language is still in settings.py as it was before., I've only added another languages there.

File: antiplag/serializers.py (hunk 3 above)
Comment (MCFreddie777) [inline antiplag/serializers.py:67]: I saw you made changes on another branch. Can you please remove it from this pull request?

**Review by MCFreddie777 — CHANGES_REQUESTED** [review]
Please remove that one change you extracted to other PR. Otherwise, it looks okay to be (except those comments in `.po` file). But maybe @OliverKanat review and final approve will be more valuable here than mine.

File: django_project/settings.py [hunk 4]
```diff
@@ -107,6 +110,11 @@
 
 LANGUAGE_CODE = "en-us"
 
+LANGUAGES = [
+    ('en-us', _('English')),
```
Comment (MCFreddie777) [inline django_project/settings.py:114]: Why do we use  `ugettext_lazy` somewhere and `ugettext` elsewhere? What is the difference between those two?

**Review by MCFreddie777 — COMMENTED** [review]
What si `antiplag/django-admin` file? It seems empty. What does it do?

File: django_project/settings.py (hunk 4 above)
Comment (MCFreddie777) [reply django_project/settings.py:114]: Makes sense

**Review by MCFreddie777 — APPROVED** [review]

**Review by OliverKanat — APPROVED** [review]

File: antiplag/serializers.py (hunk 3 above)
Comment (DeMensa123) [reply antiplag/serializers.py]: Because this file is moved to its own branch. It's not here anymore.

**Pull request merged**